flexmeasures/data/migrations/versions/4b5aa7856932_added_primary_and_secondary_color_field_.py

```diff
@@ -0,0 +1,458 @@
+"""Added primary and secondary color field to account model
+
+Revision ID: 4b5aa7856932
+Revises: 126d65cbe6b4
+Create Date: 2024-08-08 13:38:17.197805
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "4b5aa7856932"
+down_revision = "126d65cbe6b4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("asset")
+    op.drop_table("asset_type")
+    op.drop_table("weather_sensor")
+    op.drop_table("market")
+    op.drop_table("weather_sensor_type")
+    op.drop_table("market_type")
+    op.drop_table("power")
+    op.drop_table("weather")
```

Comment:
These upgrade instruction should not be part of this PR. I had expected these old tables to be already deleted, so I'm not sure why they were auto-generated, but that's a problem for another day.

Reply:
Ok, do i manually remove them or leave them be for now?

Reply:
Manually remove, as my other comment said (I think). And you can drop these tables from your dev DB as well.